int64 pressure
int64 timeout_ms
---
bool vacuumSuccess